user-interface/src/components/CaseAssignment.test.tsx

```diff
@@ -181,158 +181,161 @@ describe('CaseAssignment Component Tests', () => {
     );
   });
 
```

Comment:
Left tests commented out since the tests are good when they pass and we spent a lot of time on them.  If we can get them to pass consistently then we should enable them.  Leaving them there in the code for now.